engines/registry/app/models/access_scope.rb

```diff
@@ -88,19 +88,19 @@ def allowed_paths(system = nil)
     allowed_products = (active_products & access_policies_yml.keys)
```

Comment:
```suggestion
    allowed_product_classes = (active_products & access_policies_yml.keys)
```

also above, it should read `active_product_classes`